sinch/domains/voice/models/calls/responses.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+from sinch.domains.voice.models import Price, Destination
+
+
+@dataclass
+class GetVoiceCallResponse(SinchBaseModel):
+    from_: str
+    to: Destination
+    domain: str
+    call_id: str
+    duration: int
+    status: str
+    result: str
+    reason: str
+    timestamp: str
```

Comment:
I can, but then it could be misleading for the end user. The property name is called `timestamp`, but the type and content of this variable will be of type `datetime.datetime`.
WDYT?

Reply:
I think on the contrary that the property is wrongly named as a timestamp definition is a representation of a specific point in time, typically expressed as a number of the time that has elapsed since a reference time. Here the specification says it's in UTC format so it's definitively not a number to use: in any cases, the user will want to manipulate this data, it's better to give them a `datetime` object they won't have to parse themselves.

Reply:
I have to look at other APIs to figure out how to make this consistent across whole SDK.
I'll resolve this shortly.

Reply:
Done.
Same concept will be applied to the Verification domain - DEVEXP-354